modules/vpc-openshift/variables.tf

```diff
@@ -133,4 +133,10 @@ variable "taints" {
   }))
   description = "Set taints to worker nodes."
   default     = null
-}
\ No newline at end of file
+}
+
+variable "patch_version" {
+  description = "Updates the worker nodes with the required patch version, patch_version_fixpack_version ."
```

Comment:
Description should be 

Updates the worker nodes with the required patch version. The patch_version should be in the format: patch_version_fixpack_version.